src/shell/components/shell-view.scss

```diff
@@ -11,6 +11,11 @@
     src: url('../../../assets/selawkl.woff2') format('woff2'),
          url('../../../assets/selawkl.woff') format('woff');
 }
+@font-face {
+    font-family: 'Selawik Semibold';
+    src: url('../../../assets/selawkl.woff2') format('woff2'),
+         url('../../../assets/selawkl.woff') format('woff');
+}
```

Comment:
Is there much point to having this if its the exact same asset as `Selawik Light`? Or is the intent to get the `Semibold` font as well?